subscribie/blueprints/admin/templates/admin/add_item.html

```diff
@@ -1,86 +1,146 @@
-<legend>New Item</legend>
-    <div style="display:none">
-    {{ render_field(form.company_name, value=company.name ) }}
-    <input name="email" type="hidden" value='email@example.com' />
-    </div>
-	<dl>
-      <dt><label>Title:</label></dt>
-      <dd><input name="title-0" /></dd>
-
-      <dt><label>Primary Icon:</label></dt>
-      <dd><input type="file" name="image-0" /></dd>
-
-	  <dt><label>Requires Monthly Price:</label></dt>
-      <dd><input name="subscription-0" type="checkbox" />
-      </dd>
-
-	  <dt><label>Monthly Price:</label></dt>
-      <dd><input name="monthly_price-0" placeholder="9.99" /></dd>
-
-      <dd>
-          <p>Number of days before first monthly charge. e.g. "Cooling off period"
-          Zero is as soon as possible:</p>
-          <input type="number" name="days_before_first_charge-0" value="0"></textarea>
-      </dd>
-
-	  <dt><label>Requires Up-front Price:</label></dt>
-      <dd><input name="instant_payment-0" type="checkbox" />
-      </dd>
-
-	  <dt><label>Up-front Price:</label></dt>
-      <dd><input name="sell_price-0" placeholder="20.00" /></dd>
-    </dl>
-
-    <dt><label>Require note from customer: <em>For example, for personalisation</em></label></dt>
-      <dd><input name="note_to_seller_required-0" type="checkbox" />
-      </dd>
-      <dd>
-          <p>Message shown to buyer. e.g. "Please provide alternative contact number"</p>
-          <textarea name="note_to_buyer_message-0"></textarea>
-      </dd>
-
-        <dl>
-          <dt><legend>Selling Points:</legend></dt>
-		  <dd>
-              <input name="selling_points-0-0" placeholder="Handmade" />
-          </dd>
-		  <dd>
-              <input name="selling_points-0-1" placeholder="Top quality" />
-          </dd>
-		  <dd>
-              <input name="selling_points-0-3" placeholder="Excellent support" />
-          </dd>
-        </dl>
-</fieldset>
-  <p><input type="submit" value="Save">
-</form>
-</div><!-- end .section -->
+<main>
+  <div class="section">
+    {% if form.errors %}
+    <div class="container">
+      <div class="alert alert-danger col-md-6 mx-auto">
+        <p><strong class="alert-heading">Oops, something is missing. Please take a look at:</strong></p>
+        <ul class=errors>                                                            
+          {% for error in form.errors %}                                              
+            <li>{{ error }}</li>                                                       
+          {% endfor %}                                                                 
+        </ul> 
+      </div>                                                        
+    </div>                                                                    
+      {% endif %} 
+
+    <div class="container">
+
+      <div class="card mx-auto col-md-6">
+        <form method="post" action="#" enctype="multipart/form-data">
+          {{ form.csrf_token }}
+
+          <div style="display:none">
```

Comment:
@chrisjsimpson is it needed here?

Reply:
@Svyatoslav-Polishchuk The form.csrf_token? Yes I believe so :+1:

Reply:
I meant the hidden div with <input name="email" type="hidden" value='email@example.com' />
@chrisjsimpson

Reply:
@Svyatoslav-Polishchuk ah. yes it is at the moment. I need to resolve that at some point.